app/src/main/res/layout/fragment_contribution_diff_detail.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please make this view scrollable because the content is cut off in landscape mode.